modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -247,3 +247,24 @@ mirror:
  additionalImages:
    - name: registry.redhat.io/ubi9/ubi:latest
 ----
+
+[discrete]
+[id="oc-mirror-image-set-examples-blocked-images_{context}"]
+== Use case: Including the `blockedImages` parameters
```

Comment:
I won't hold up merge review for this, but parameters should be singular. If you want to fix it in a later PR, feel free, otherwise I'll make note to change it when I take a closer look at oc-mirror docs after GA.
```suggestion
== Use case: Including the `blockedImages` parameter
```